enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/util/BadgeList/AddBadgeListItem/AddBadgeListItem.tsx

```diff
@@ -1,10 +1,10 @@
-import { Badge, type BadgeProps } from "@mantine/core";
 import { type HTMLAttributes, type Ref, forwardRef } from "react";
 
 import CS from "metabase/css/core/index.css";
-import { ActionIcon, Icon } from "metabase/ui";
+import { ActionIcon, Badge, Icon } from "metabase/ui";
 
-type BadgeListItemRootProps = BadgeProps & HTMLAttributes<HTMLDivElement>;
+type BadgeListItemRootProps = React.ComponentProps<typeof Badge> &
```

Comment:
should we just export the props type like other components? I think I'd be more consistent that way

https://github.com/metabase/metabase/blob/7da3631d787c1eb393512c34909864f94b479208/frontend/src/metabase/ui/components/utils/Box/index.ts#L1-L2